observability/observability.go

```diff
@@ -75,7 +45,7 @@ type configOut struct {
 	Config []config.ExportedConfig `group:"config,flatten"`
 }
 
-func provideConfig() configOut {
+func exportConfig() configOut {
```

Comment:
Other package export config func name is provideConfig, unified? Which one should we use?

Reply:
provideConfig. As this is a private function, we can always change it at any point.